inspirehep/modules/search/static/js/search/app.js

```diff
@@ -22,9 +22,13 @@
  */
 
 require([
+    'angular',
+    'angular-sanitize',
+    'angular-ui-bootstrap',
     'invenio-search',
     'inspirehep-search',
-    'inspirehep'
+    'inspirehep',
+    'ngclipboard'
```

Comment:
Instead of adding it here, it is better if we add it in the `settings.js` close to the library that really needs it as a dependency - https://github.com/inspirehep/inspire-next/blob/invenio3/inspirehep/modules/theme/static/js/settings.js#L69